share/errmsg-utf8.txt

```diff
@@ -18983,6 +18983,9 @@ ER_RDB_TTL_DURATION_FORMAT
 ER_PER_INDEX_CF_DEPRECATED
   eng "The per-index column family option has been deprecated."
 
+ER_REDO_ENCRYPTION_CANT_BE_CHANGED
```

Comment:
This is not a MyRocks-specific message - wrong file section then?

Reply:
Right, moved.